onnxruntime/contrib_ops/cpu/fusedmatmul.cc

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: Can the filename be consistent with the other fused_* files (fused_conv, fused_gemm, etc)?